config/suppressions-xpath.xml

```diff
@@ -7,18 +7,18 @@
 <suppressions>
   <!-- Tone down the checking for test code -->
   <suppress-xpath checks="NPathComplexity"
-                   query="/CLASS_DEF[./IDENT[@text='XdocsPagesTest']]
+                   query="//CLASS_DEF[./IDENT[@text='XdocsPagesTest']]
```

Comment:
I don't see why these have to change if the suppression still works. This, as an example, will never be a sub-class as it is a test class.

Reply:
Actually I think I see why they need to change. They either need to be `//CLASS_DEF` or become `/COMPILATION_UNIT/CLASS_DEF`. I assume the later will be a bit speedier in finding a match since its not `//`?

Reply:
@romani preferred `//` over full path for readability.

Reply:
Ok, then we can ignore this. I will mark the 2 resolved.